code/modules/mapping/ruins.dm

```diff
@@ -52,7 +59,7 @@
 	return center
 
 
-/proc/seedRuins(list/z_levels = null, budget = 0, whitelist = /area/space, list/potentialRuins)
+/proc/seedRuins(list/z_levels = null, budget = 0, whitelist = list(/area/space), list/potentialRuins)

```

Comment:
Why is `whitelist` a list var? Despite its name it seems to actually be an area variable in functionality.

Reply:
It is the areas that a ruin can be placed, it is a list so you can have multiple areas that a ruin can be allowed to be placed in, this is necessary for ruin generation in z levels with multiple areas or multi-z ruins where the ruin is placed on a top and bottom z level with different areas.